src/sif/pedestriancost.cc

```diff
@@ -42,6 +42,9 @@ constexpr float kDefaultSpeedWheelchair = 4.0f; // 2.5  MPH  TODO
 constexpr float kDefaultStepPenaltyFoot = 30.0f;        // 30 seconds
 constexpr float kDefaultStepPenaltyWheelchair = 600.0f; // 10 minutes
 
+// Penalty to take elevator
+constexpr float kDefaultElevatorPenalty = 5.0f; // 5 seconds
```

Comment:
this seems quite cheap!

on a serious note though, if we want to keep the changes to costing in this pr we should have a test that exorcises them in your test suite. do you mind adding a test? maybe you want to set a super high penalty in the request so that a person takes the stairs instead of the elevator and your test can just show  you took the stairs when you set the penalty but you take the elevator otherwise? or did you want to save all that for another pr?

Reply:
Are you talking about adding to my gurka test or a different test? It will be great if you can help with the tests. If you feel 5s is too cheap, I'm fine with whatever number you feel is more reasonable. Another option (as you said) is to save them for my next PR for maneuvers - which I can happily take. Let me know what you would prefer!

Reply:
oh yeah i would just add it to the existing tests, that way we can keep all the indoor routing tests right there, data, and algorithm righ tnext to each other and easy to find.

regarding the number im fine with 5 seconds for now, it just seemed a little small but i was really just kidding around. and yeah i can happily make a simple test in there!

Reply:
Hi @kevinkreiser, I propose to move all the changes (and tests) in pedestriancost.cc to a second PR, so that we can merge this PR first, if this test is the only blocker at this point. In this way, you can take your time with the tests while I am working on another PR for indoor routing maneuvers/narrations. What do you think?

Reply:
haha im literally writing the test right now! i'll put another hour into it but if it doesnt finish after that then i agree. yes that is the only blocker at hte moment. i just need to give it a final review after adding the test. sorry for the delay hard to find the time these past couple of nights

Reply:
ah good thing i wrote these tests, i caught a small typo that messed up the access for the edges and nodes! ill push it up shortly!

Reply:
ah it seems you have your fork configured such that i cannot push to it. would you mind changing that at least temporarily?

Reply:
oh it seem the github gui will let do it that way. since its only 2 files ill just do that. one sec!

Reply:
ok just pushed!